src/sentry/incidents/models.py

```diff
@@ -368,3 +368,41 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_alertruletriggerexclusion"
         unique_together = (("alert_rule_trigger", "query_subscription"),)
+
+
+class AlertRuleTriggerAction(Model):
+    """
+    This model represents an action that occurs when a trigger is fired. This is
+    typically some sort of notification.
+    """
+
+    __core__ = True
+
+    # Which sort of action to take
+    class Type(Enum):
+        EMAIL = 0
+        PAGERDUTY = 1
+        SLACK = 2
```

Comment:
Will this be something that expands as new integrations are added? Should this be a hook/signal that plugins/integrations can implement?

Reply:
It might. For the moment I'm happy to be explicit, but we might want plugins to be able to automatically hook into this in the future.